### ISSUE TITLE
Analyzers prototyping - Bootstrap adjustments

### ISSUE THREAD

**Issue by JanKrivanek** [dotnet/msbuild#9734]
### Context

#9627

Analyzers.UnitTests need bootstrap to run an e2e integration test - for an accurate developer experience, all dev changes should be reflected in bootstrap properly.
* Analyzers project should be referenced by bootstrap and output assembly should be placed to bootstrap outputs (both conditions should be already met)
* Analyzers.UnitTests build should lead to building changes in any changed project and placing them into bootstrap location - via the bootsrap project reference - this doesn't seem to work correctly currently.

**Comment by JanKrivanek** [comment]
possibly inputs/outputs issue with bootstrap project and hance false u2d?

```
========== Build: 0 succeeded, 0 failed, 11 up-to-date, 0 skipped ==========
```
When building bootstrap after building analyzers (from VS, wasn't yet reproing on commandline)

**Comment by ladipro** [comment]
Fixed in 455cc7b1f80ee85695cac33ad19b50ffdc678051 by disabling build acceleration for the bootstrap project.
